meson.build

```diff
@@ -3,7 +3,10 @@ project('lite-xl',
     version : '2.0.3',
     license : 'MIT',
     meson_version : '>= 0.42',
-    default_options : ['c_std=gnu11']
+    default_options : [
+        'c_std=gnu11',
+        'wrap_mode=nofallback'
```

Comment:
`wrap_mode` now defaults to `nofallback`

Reply:
`nodownload` may be a safer default. The difference occurs when someone manually does `meson subprojects download foo`.

Then the current PR as is will still require overriding --wrap-mode on the command line, while `nodownload` says "you downloaded it by hand, thus clearly you want to use it".

The subprojects won't generally be downloaded, unless the user manually did that. There is also the possibility of uploading source tarballs to Github Releases using `meson dist --include-subprojects`, but most people don't use that... it is of interest to some people who for "lawyer reasons" want to publish source code that has everything and the kitchen sink included.

But either one is fine and fulfills the primary objective of making wrap fallbacks opt in.

Reply:
Problem is, if for whatever reason the fallbacks are downloaded (lets say someone forces fallbacks to create a static build) every following build will use the fallbacks, which seems undesired, if I understand Franko correctly